### ISSUE TITLE
TMTIntegrator cannot find reference channel using default settings and correct tag

### ISSUE THREAD

**Issue by gigi-mt** [Nesvilab/FragPipe#621]
Hello FragPipe team.
First of all thank you for the user friendly tool that combines all the great software into one interface!

I ran FragPipe using two sets of TMT11plexes, with each 23 fractions and a pooled sample as reference channel, tagged as pool01 or pool02 in the respective annotation.txt. 
MSFragger, Philosopher, Percolator all finished without errors. 
TMTIntegrator stopped giving following message:
         Process 'TmtIntegrator' finished, exit code: 1
         Process returned non-zero exit code, stopping saying it could not find the reference channel. 

I used default settings for setting up the search, and the annotation.txt has the correct channel and tag for the reference sample. 
The log is attached. 

What am I missing? 

Thanks for your help,
Georgia

[log_2022-03-09_23-59-25.txt](https://github.com/Nesvilab/FragPipe/files/8224066/log_2022-03-09_23-59-25.txt)

**Comment by fcyu** [comment]
Hi Georgia,

I see that the reference channel tag was set to `pool`. I am not sure if TMT-Integrator can recognize `pool01` and `pool02` in your annotation.txt. Need @anesvi or Hui-Yin's help.

Best,

Fengchao

**Comment by gigi-mt** [comment]
Thank you for the quick reply.

The PSM.tsv has wrong TMT-labels for channel 131C and 131N.
I double-checked the respective annotation.txt and saw that the 131C assignment was missing for that file.

I am re-running the search.

Is there a function to partially re-process the search, the way it is for MQ or proteome discoverer?

Thanks,
G

**Comment by anesvi** [comment]
Yes, you only need to rerun the Isobaric labeling (with TMT-Integrator) tab. Uncheck MSFragger and everything in the Validation tab and click run

**Comment by gigi-mt** [comment]
Marveouls! Great software Alexey!

G
________________________________
